src/Tests/TestComponentCSharp/Directory.Build.props

```diff
@@ -1,10 +1,15 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" >
 
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+    <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 
-  <!--Only need the WinUI winmd files for compiling idl, so stub out Xaml targets-->
-  <Target Name="MarkupCompilePass1" />
-  <Target Name="MarkupCompilePass2" />
-  <Target Name="Prep_ComputeProcessXamlFiles" />
+    <PropertyGroup>
+        <!--Avoid UniversalApiContract type not found error-->
+        <CppWinRTMergeNoValidate>true</CppWinRTMergeNoValidate>
```

Comment:
Should this go away too?

Reply:
no, this has to remain - we use UniversalApiContract in our tests for SupportedOSPlatform